lib/src/util/io.dart

```diff
@@ -18,6 +18,23 @@ const _newline = 0xA;
 /// The ASCII code for a carriage return character.
 const _carriageReturn = 0xD;
 
+/// The default line length for output when there isn't a terminal attached to
+/// stdout.
+const _defaultLineLength = 200;
```

Comment:
I think the fallback should get called out in the commit message.

Why was 200 chosen instead of the old value of 100?

Reply:
> I think the fallback should get called out in the commit message.

Done.

> Why was 200 chosen instead of the old value of 100?

100 was chosen to approximate the average terminal width. Now that we detect the actual width for real terminals, I want to increase the default because it's more likely to come into play in non-width-constrained environments.